src/App.js

```diff
@@ -1,24 +1,74 @@
-import logo from './logo.svg';
+import React, { useEffect } from 'react';
 import './App.css';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+// Import the necessary components
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import SignInPage from './pages/signin/SignIn';
+import Layout from './components/layout/Layout';
+import Main from './routes/Home';
+import Delete from './routes/Delete';
+import AddNew from './routes/AddNew';
+import MyReservations from './routes/MyReservations';
+import Reserve from './routes/Reserve';
+import ItemDetail from './routes/ItemDetail';
+import { usersSelector } from './redux/store';
+import createUser from './redux/user/thunk';
+import LoadingScreen from './components/conditions/LoadingScreen';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this main "App" functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Salimer/Final-Capstone-Front-End/assets/57421823/9407fd0b-b614-4bbb-9a38-c2f8ead581d2)



That way, your code will meet the latest standards. 💪